### PR TITLE
EmailAuthPage 생성, Register/LoginPage 이동 기능입니다.

### PR DESCRIPTION
**Pull request by chepchep2**
EmailAuthPage 생성
- 알려주신 코드 참고해서 EmailAuthPage 생성했습니다.
- Register/Login Page 이동 가능 기능입니다.
- [구현결과](https://imgur.com/Im78Ph1)

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#33** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/33?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#32** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/32?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#31** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/31?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#30** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/30?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#29** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/29?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#28** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/28?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#27** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/27?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#26** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/26?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#25** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/25?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/24?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#23** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/23?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#22** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/22?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#21** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/21?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/20?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#19** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/19?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#18** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/18?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/page/login_page.dart [hunk 1]
```diff
@@ -49,25 +55,28 @@ class _LoginPageState extends State<LoginPage> {
...
-                  const Row(
+                  Row(
                     children: [
-                      Text(
-                        '회원가입',
-                        style: TextStyle(
-                          fontSize: 28,
-                          fontWeight: FontWeight.bold,
-                          color: AppColor.textPrimary20,
+                      GestureDetector(
```
Comment (flowkater) [inline lib/page/login_page.dart:60]: 이 부분을 상위 EmailAuthPage Header 로 빼면 어떨까요?

File: lib/page/register_page.dart [hunk 2]
```diff
@@ -67,19 +70,13 @@ class _RegisterPageState extends State<RegisterPage> {
...
                         child: const Text(
                           '로그인',
                           style: TextStyle(
-                            fontSize: 28,
-                            fontWeight: FontWeight.bold,
-                            color: AppColor.textPrimary20,
-                          ),
+                              fontSize: 28,
+                              fontWeight: FontWeight.bold,
+                              color: AppColor.textPrimary20),
```
Comment (flowkater) [inline lib/page/register_page.dart:79]: 항상 ,(comma)로 끝내기

File: lib/page/login_page.dart (hunk 1 above)
Comment (chepchep2) [reply lib/page/login_page.dart:60]: 수정해보곘습니다.

File: lib/page/register_page.dart (hunk 2 above)
Comment (chepchep2) [reply lib/page/register_page.dart:79]: 네 알겠습니다. 신경 쓰겠습니다.

**Comment by chepchep2** [comment]
### Merge activity

* **Mar 17, 8:12 PM EDT**: @chepchep2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/26).
* **Mar 17, 8:36 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/26) rebased this pull request as part of a merge.
* **Mar 17, 8:39 PM EDT**: @chepchep2 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/26).

**Pull request merged**